src/main/java/Chandler.java

```diff
@@ -0,0 +1,113 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Chandler {
+
+    public static final String LINE_DIVIDER = "____________________________________________________________";
+    public static final String OUTPUT_INDENTATION = "    ";
+
+    public static void main(String[] args) {
+
+        System.out.println(LINE_DIVIDER);
+        System.out.println("Hello! I'm Chandler. Your personal sarcastic task manager.");
+        Scanner myObj = new Scanner(System.in);
+        System.out.println("So, what can you do for me? :)");
+        System.out.println(LINE_DIVIDER);
+
+        String input = myObj.nextLine();
+        List<Task> list = new ArrayList<>();
+
+
+        for(int listSize = 1; !input.equals("bye"); input = myObj.nextLine()) {
+            try{
+                System.out.println(LINE_DIVIDER);
+                String[] words = input.split("\\s+");
+                String inputCommand = words[0];
+
+                if (words.length <= 1 && !inputCommand.equals("list") && !inputCommand.equals("bye") && !inputCommand.equals("mark") && !inputCommand.equals("unmark")) {
```

Comment:
This is a fairly complex code statement. To improvise the readability for the user, you might want to break this down to simpler statements and use boolean variables to track each sub-expression.